### PR TITLE
x11-misc/fqterm: fix #272

### LINKED ISSUE #272: x11-misc/fqterm has moved to here https://github.com/mytbk/fqterm
So, why not update it?

### PR DESCRIPTION
**Pull request by riaqn**
The development of fqterm has moved to https://github.com/mytbk/fqterm,
this commit fix the source uri in ebuilds, along with a release version.

### CODE REVIEW

**Comment by JiajunW** [comment]
版本号不对。应该是 fqterm-0.9.8.3-r4 吧

**Comment by riaqn** [comment]
@JiajunW 
http://devmanual.gentoo.org/general-concepts/ebuild-revisions/
revision应该是包维护用的版本号吧, 原作者用的版本号与这个规则冲突, 所以我改成了0.9.8.3.4.

**Comment by JiajunW** [comment]
哦那版本号也应该是 fqterm-0.9.8.3.r4

**Comment by riaqn** [comment]
@JiajunW 
http://devmanual.gentoo.org/ebuild-writing/file-format/index.html#file-naming-rules
那样emerge会报错..

**Comment by JiajunW** [comment]
的确是…………

那我觉得应该还是 -r4 这种啊，符合 http://devmanual.gentoo.org/general-concepts/ebuild-revisions/ 这里说的情况：

> Ebuilds should have their -rX incremented whenever a change is made which will make a substantial difference to what gets installed by the package — by substantial, we generally mean "something for which many users would want to upgrade". This is usually for bugfixes.

实际上 https://github.com/mytbk/fqterm/releases 看这里的说明都是小的 fix

File: x11-misc/fqterm/fqterm-0.9.8.3.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="4"
```
Comment (JiajunW) [inline x11-misc/fqterm/fqterm-0.9.8.3.4.ebuild]: 没有特殊情况用 5

**Pull request merged**